src/main/java/org/apache/sling/feature/extension/apiregions/api/config/PropertyDescription.java

```diff
@@ -457,4 +465,33 @@ public PlaceholderPolicy getPlaceholderPolicy() {
     public void setPlaceholderPolicy(final PlaceholderPolicy policy) {
         this.placeholderPolicy = policy == null ? PlaceholderPolicy.DEFAULT : policy;
     }
+
+	/**
+	 * Get the placeholder regex
+	 * @return the placeholder regex or {@code null}
+	 */
+    public String getPlaceholderRegex() {
```

Comment:
Please add a @since tag for the new methods

Reply:
Thanks for the suggestion, @cziegeler - I've done this in my latest commit.